geth/txqueue/fake/mock.go

```diff
@@ -0,0 +1,91 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Please add a proper entry to `mock` target: https://github.com/status-im/status-go/blob/develop/Makefile#L101-L105